docs/user/adding-new-metrics.md

```diff
@@ -67,19 +67,9 @@ If the metric is still needed after its expiration date, it should go back for [
 
 ### When should Glean automatically clear the measurement?
 
-The `lifetime` parameter of a metric defines when its value will be cleared. There are three lifetime options available:
+The `lifetime` parameter of a metric defines when its value will be cleared. There are three lifetime options available, [`ping`, `application` and `user`](metrics-parameters.html).
```

Comment:
While generally I don't like duplication in the docs, on this one I'm kind of :/  The reader really needs to click on that link and come back to understand anything about what immediately follows.  I think I'd prefer to have the long discussion here and shorten the reference documentation to just the bare bones and link back here for the details.

Reply:
I agree. Is there a server-side inlining step we could use to embed the same content in both places, perhaps?

Reply:
There is:

```
{{#include ../path/to/file.md}}
```

Care needs to be taken for relative links in the included file though.

Reply:
Oh, nice. Let me try this, thanks @badboy